.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/Roslyn"]
+	path = src/Roslyn
+	url = https://github.com/nguerrera/Roslyn
```

Comment:
See https://github.com/dotnet/roslyn/compare/master...nguerrera:il-analysis for experimental changes to roslyn depended upon so far.